src/sentry/api/serializers/models/integration.py

```diff
@@ -30,8 +34,10 @@ def serialize_provider(provider):
 
 
 @register(Integration)
-class IntegrationSerializer(Serializer):
-    def serialize(self, obj, attrs, user):
+class IntegrationSerializer(Serializer):  # type: ignore
+    def serialize(
+        self, obj: Integration, attrs: Mapping[str, Any], user: User, **kwargs: Any
+    ) -> MutableMapping[str, Any]:
```

Comment:
We could potentially have this be `[str, JSONData]`. At the moment `JSONData` is just `Any`, but at least it's a little  more descriptive of what we intend to return here

Reply:
This is great, thanks 👍🏽